release_dashboard/test/fakes/services/fake_conductor.dart

```diff
@@ -6,25 +6,32 @@ import 'package:conductor_core/src/proto/conductor_state.pb.dart';
 import 'package:conductor_ui/services/conductor.dart';
 import 'package:file/file.dart';
 import 'package:file/memory.dart';
+import 'package:flutter/material.dart';
 import 'package:platform/platform.dart';
 
+import '../fake_next_context.dart';
 import '../fake_start_context.dart';
 
 /// Fake service class for using the conductor in a fake local environment.
 ///
 /// When [fakeStartContextProvided] is not provided, the class initializes
 /// a [createRelease] method that does not throw any error by default.
 ///
+/// When [fakeNextContextProvided] is not provided, the class initializes
+/// a [conductorNext] method that does not throw any error by default.
+///
 /// [testState] parameter accepts a fake test state and passes it to the
 /// release dashboard.
 class FakeConductor extends ConductorService {
   FakeConductor({
     this.fakeStartContextProvided,
+    this.fakeNextContextProvided,
     this.testState,
   });
 
   final FakeStartContext? fakeStartContextProvided;
-  final ConductorState? testState;
+  FakeNextContext? fakeNextContextProvided;
+  ConductorState? testState;
```

Comment:
making `testState` a field allows to modify `testState` after the class initialization.